src/main/webapp/WEB-INF/tags/instructor/courseStudentDetails/studentInformationTable.tag

```diff
@@ -0,0 +1,201 @@
+                        </td>
+                        <td>
+                            <input class="visibilityCheckbox giverCheckbox"
+                                   type="checkbox" value="<%= CommentParticipantType.COURSE %>">
+                        </td>
+                        <td>
+                            <input class="visibilityCheckbox recipientCheckbox"
+                                   type="checkbox" value="<%= CommentParticipantType.COURSE %>">
+                        </td>
+                    </tr>
+                    <tr>
+                        <td class="text-left">
+                            <div data-toggle="tooltip" data-placement="top" title=""
+                                 data-original-title="Control what instructors can view">
+                                Instructors
+                            </div>
+                        </td>
+                        <td>
+                            <input class="visibilityCheckbox answerCheckbox"
+                                   type="checkbox" value="<%= CommentParticipantType.INSTRUCTOR %>">
+                        </td>
+                        <td>
+                            <input class="visibilityCheckbox giverCheckbox"
+                                   type="checkbox" value="<%= CommentParticipantType.INSTRUCTOR %>">
+                        </td>
+                        <td>
+                            <input class="visibilityCheckbox recipientCheckbox"
+                                   type="checkbox" value="<%= CommentParticipantType.INSTRUCTOR %>">
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+        <textarea class="form-control" rows="6" placeholder="Enter your comment here ..." style="margin-bottom: 15px;"
+                  name="<%= Const.ParamsNames.COMMENT_TEXT %>" id="commentText"></textarea>
+        <div style="text-align: center;">
+            <input type="submit" class="btn btn-primary" id="button_save_comment" value="Save"> 
+            <input type="button" class="btn btn-default" id="button_cancel_comment" value="Cancel">
+            <input type="hidden" name=<%=Const.ParamsNames.COURSE_ID%> value="${studentInfoTable.course}">
+            <input type="hidden" name=<%=Const.ParamsNames.STUDENT_EMAIL%> value="${studentInfoTable.email}">
+            <input type="hidden" name=<%=Const.ParamsNames.RECIPIENT_TYPE%> value="<%= CommentParticipantType.PERSON %>">
+            <input type="hidden" name=<%=Const.ParamsNames.RECIPIENTS%> value="${studentInfoTable.email}">
+            <input type="hidden" name=<%=Const.ParamsNames.COMMENTS_SHOWCOMMENTSTO%> value="">
+            <input type="hidden" name=<%=Const.ParamsNames.COMMENTS_SHOWGIVERTO%> value="">
+            <input type="hidden" name=<%=Const.ParamsNames.COMMENTS_SHOWRECIPIENTTO%> value="">
```

Comment:
Ah, oops.
...I wonder exactly what the person who put these here intended to do.
And whether it worked.